From c9209fa0f474d41bc5ecf2b988ab404123038c1b Mon Sep 17 00:00:00 2001
From: Brett Warren <brett.warren@arm.com>
Date: Tue, 3 Nov 2020 13:34:26 +0000
Subject: [PATCH] pmf.h: made PMF_STOTE_ENABLE pass -Wtautological

When compiling with clang, PMF_STORE_ENABLE triggers
-Wtautological-constant-compare. To mitigate, the definition
is modified cosmetically to not trigger this error.

Upstream-Status: Pending
Signed-off-by: Brett Warren <brett.warren@arm.com>
---
 include/lib/pmf/pmf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/lib/pmf/pmf.h b/include/lib/pmf/pmf.h
index df7c9ff31..baa2dfd60 100644
--- a/include/lib/pmf/pmf.h
+++ b/include/lib/pmf/pmf.h
@@ -25,7 +25,7 @@
 /*
  * Flags passed to PMF_REGISTER_SERVICE
  */
-#define PMF_STORE_ENABLE	(1 << 0)
+#define PMF_STORE_ENABLE	1
 #define PMF_DUMP_ENABLE		(1 << 1)
 
 /*
-- 
2.17.1

